### PR TITLE
added a few more drawImage function overloads

### PR DESCRIPTION
**Pull request by bbernhard**
* The existing drawImage function determined the image type by looking
at the file ending. However, that only works in case the image that's going
to be displayed has a file ending. This changeset adds the possibility
to explicitly specify the format (BMP, JPG, PNG).

* Added the possibility to specify the position (Center, TopLeft,
BottomLeft, TopRight, BottomRight) of the image on the screen. This
function overload is purely for convenience and makes it easier to
position the image on the screen without needing to specify the x and y
pos. Currently, only jpg images fetched from web can be positioned that
way; for every other image format and source (e.g SD card),
the appropriate functionality still needs to be implemented.

As discussed in #48, I've tried to implement a few functions that I personally would find pretty convenient. I've only implemented the functionality that I need, so in order to be feature complete, there are definitely some changes needed. Not sure if the API is the way you imagined it. Feedback & suggestions are highly appreciated.

### CODE REVIEW

**Review by nitko12 — APPROVED** [review]
Good to merge, once merged, I'll implement these TODOs.
Should be merged to revision/dev branch tho.

**Comment by davidzovko** [comment]
Thanks @bbernhard 
Changed branch to revision as suggested and approved merge.

**Pull request merged**
